docs/manual.rst

```diff
@@ -263,9 +275,23 @@ Once a torrent completes checking and moves into a diffferent state, the next in
 line will be started for checking.
 
 Any torrent added force-started or force-stopped (i.e. the auto managed flag is
-_not_ set), will not be subject to this limit and they will all check
+*not* set), will not be subject to this limit and they will all check
 independently and in parallel.
 
+In order to add a torrent, have its resume data and potentially files checked
```

Comment:
Add commas or parentheses around "and potentially files"? Not sure which is more correct.
resume data, and potentially files, checked
resume data (and potentially files) checked